### DISCUSSION TITLE
dcoker issue

### DISCUSSION THREAD

**Discussion by dragelecpc** [custompbx/custompbx#3] [General]
Hi, i deploy a testing by using your docker file. Docker can up but cannot login, it just show the login page without error (and said no websocket). can you please recheck the docker ?

**Comment by zusrut** [comment]
make sure you replaced host in `docker-compose.yml` file in line `- WS_BACKGROUND_OVERRIDE=wss://127.0.0.1:8080/ws` to correct one. it should be the host your custompbx running and your browser will try to connect via websocket. what exactly URL it tries to reach - you can see in browser console error messages. 
then try to open this websocket url in browser but replacing `wss` to `https` like `https://127.0.0.1:8080/ws` and allow self signed certificate if browser asked
